mmcls/models/backbones/convvit.py

```diff
@@ -0,0 +1,381 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Sequence, Union, Tuple, Optional
+
+import torch
+import torch.nn as nn
+import numpy as np
+from mmcv.cnn.bricks.transformer import PatchEmbed
+from mmcv.cnn.bricks import DropPath, build_norm_layer, build_activation_layer
+from mmcv.runner.base_module import BaseModule
+
+from ..utils import to_2tuple
+from ..builder import BACKBONES
+from .base_backbone import BaseBackbone
+from .vision_transformer import TransformerEncoderLayer
+
+
+class ConvMlp(BaseModule):
+    """Implements the Mlp part for ConvBlock. The main difference is ConvMlp
```

Comment:
``The main difference is ConvMlp implies N, C, H, W input format.`` is confused, since it does not specifiy the module ``ConvMlp`` is compared to.